services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslMovementSpec.groovy

```diff
@@ -38,12 +39,12 @@ class IslMovementSpec extends BaseSpecification {
         def isl = topology.islsForActiveSwitches.find {
             it.getAswitch()?.inPort && it.getAswitch()?.outPort
         }
-        assert isl
 
         and: "A non-connected a-switch link"
         def notConnectedIsl = topology.notConnectedIsls.find {
             it.srcSwitch != isl.srcSwitch && it.srcSwitch != isl.dstSwitch
```

Comment:
Maybe it'd make sense to use safe navigation here if you don't assert ISL anymore?

Reply:
reworked this place